src/rest_framework_dso/fields.py

```diff
@@ -124,9 +130,39 @@ def get_related_list_ids(self, instances) -> list:
             related_mgr = getattr(instance, self.attname, None)
             if related_mgr is None:
                 continue
-            ids |= set(r.pk for r in related_mgr.all())
+            source_is_temporal = self.parent_model.is_temporal()
+            target_is_temporal = related_mgr.model.is_temporal()
+            if not source_is_temporal and target_is_temporal:
+
+                ids |= set(self._get_temporal_ids(instance, related_mgr))
+            else:
+                ids |= set(r.pk for r in related_mgr.all())
         return list(ids)
 
+    def _get_temporal_ids(self, instance, related_mgr):
+        (
+            identificatie_fieldname,
+            volgnummer_fieldname,
+        ) = related_mgr.model._table_schema.identifier
+        source_field_name = related_mgr.source_field_name
+        source_id = instance.id
+        tussentabel_filter_params = {source_field_name: source_id}
+        target_id_field = f"{related_mgr.target_field_name}_id"
+        tussentabel_items = [
+            getattr(item, target_id_field, None)
+            for item in related_mgr.through.objects.filter(**tussentabel_filter_params)
+        ]  # de items uit de tussen tabel.
```

Comment:
```suggestion
        tussentabel_items = related_mgr.through.objects.filter(**tussentabel_filter_params).values_list(target_id_field, flat=True)
```

Kan dit ook? Of werkt dat niet bij ieder veldtype?

Reply:
Dat zou moeten werken, ik heb het aangepast. Mooie performance winst.